internal/routes/all_routes.go

```diff
@@ -1,15 +1,24 @@
 package routes
 
 import (
+	"net/http"
+
 	s "github.com/RHEnVision/provisioning-backend/internal/services"
 	"github.com/go-chi/chi/v5"
 )
 
 func SetupRoutes(r *chi.Mux) {
+	r.Get("/ping", s.StatusService)
+	r.Mount("/api/provisioning", apiRouter())
```

Comment:
Where this comes from? Is this a SRE/k8s requirement? If not, I would not include 'provisioning' prefix, 'api' is okay why not, but I see no value in this.

Reply:
It is required by SRE.

Reply:
Yeah, our app is mounted on this prefix and all our routes comming from k8s are under this prefix